OpenRA.Mods.Common/Traits/Mobile.cs

```diff
@@ -714,15 +383,16 @@ public void EnteringCell(Actor self)
 
 			var notifiers = actors.SelectMany(a => a.TraitsImplementing<INotifyCrushed>().Select(t => new TraitPair<INotifyCrushed>(a, t)));
 			foreach (var notifyCrushed in notifiers)
-				notifyCrushed.Trait.WarnCrush(notifyCrushed.Actor, self, Info.Crushes);
+				notifyCrushed.Trait.WarnCrush(notifyCrushed.Actor, self, Info.LocomotorInfo.Crushes);
 		}
 
 		public void FinishedMoving(Actor self)
 		{
 			// Need to check both fromCell and toCell because FinishedMoving is called multiple times during the move
-			// and that condition guarantees that this only runs when the unit has finished landing.
-			if (fromCell.Layer != CustomMovementLayerType.Jumpjet && toCell.Layer != CustomMovementLayerType.Jumpjet && jumpjetToken != ConditionManager.InvalidConditionToken)
-				jumpjetToken = conditionManager.RevokeCondition(self, jumpjetToken);
+			// and that condition guarantees that this only runs when the unit has finished any potential layer transition.
```

Comment:
This comment doesn't make sense to me anymore.  Was I referring to the if-statement condition or the condition-manager condition?  If the latter then this line of the comment should be removed.